src/color.ts

```diff
@@ -76,7 +76,7 @@ import {hslToRGBA} from "@daybrush/utils";
 console.log(hslToRGBA([150, 0.5, 0.4]));
 // [51, 153, 102, 1]
 */
-export function hslToRGBA(hsl: number[]): number[] {
+export function hslToRGBA(hsl: readonly [number, number, number, number?]): [number, number, number, number] {
```

Comment:
@xiaoxiangmoe 

The return type is fine, but changing the color type of the argument is a very dangerous behavior.